### ISSUE TITLE
Black screen - No Verbose nor OpenCore

### ISSUE THREAD

**Issue by billabongbruno** [pnapt/ACER-Nitro5-AN515-52-Opencore#1]
Hey, there.
How are you?

First of all, I would like to thank you for all your work.
It is astonishing work indeed, from what I can tell by the screenshots.

I myself have lost countless hours to try and get my EFI folder up and running flawless (apart from dGPU and HDMI, of course).

I had everything running flawlessly up until 2 months ago, including Wifi and Bluetooth, but for some reason Bluetooth stopped working for some reason and I cannot, for the life of me, get it going.

I am running OpenCore 0.9.1 with the latest Kexts on Ventura 13.3.1 and I decided to give your EFI a shot, using my SMBIOS info.

However, I cannot get it to boot at all, even after settings _hideverbose_ to _false_ and ensuring the _-v_ argument is present in _boot-args_.

I just get a black screen and never even get to see verbose or OpenCore.

Is there something that I'm doing wrong?
Would you be willing to help?

I can share my EFI, if you want, seeing as we took different approaches and something might be of use to you.

The only thing that I see different is that my dGPU reports as a GTX 1050 (non-TI) under Windows 11 22H2, so I'm not sure that's the issue.

Can you shed some light on this matter?

Thank you in advance,
Bruno.

**Comment by pnapt** [comment]
What is your smbios info ? It should be MacBookPro14,3 
and for the wifi you should use 
https://github.com/OpenIntelWireless/itlwm/releases , v.2.2.0 alpha 
If you update your efi and kext with ocat https://github.com/ic005k/OCAuxiliaryTools
Do not tick on `AirportItlwm.kext` It will download v.2.1.0 instead and it not work on ventura

Sorry for late reply I just log back to github

**Comment by pnapt** [comment]
For Bluetooth 
Use IntelBTPatcher.kext & IntelBluetoothFirmware.kext + BlueToolFixup.kext
do not put IntelBluetoothInjector.kext
this will make a Bluetooth work but also took a system a bit long to boot

**Comment by pnapt** [comment]
What is your smbios info ? It should be MacBookPro14,3 if you use my efi
because of the `USBMap.kext` is set target to this smbios

and for the wifi you should use 
https://github.com/OpenIntelWireless/itlwm/releases , v.2.2.0 alpha 
If you update your efi and kext with ocat https://github.com/ic005k/OCAuxiliaryTools
Do not tick on `AirportItlwm.kext` It will download v.2.1.0 instead which is not work on ventura

For Bluetooth 
follow 
https://openintelwireless.github.io/IntelBluetoothFirmware/FAQ.html#what-does-this-kext-do
In short 
Use IntelBTPatcher.kext & IntelBluetoothFirmware.kext + BlueToolFixup.kext
do not put IntelBluetoothInjector.kext
this will make a Bluetooth work but also took a system a bit long to boot

ofc "I just get a black screen and never even get to see verbose or OpenCore."
Yeah That's a very common problem if you dl pre-build efi from the internet
a laptop with the same model sometime in different region some component isn't the same 
I had got the same problem too in good old clover bootloader days

P.S.#1 I do not have the device/own the device This is my friend laptop 
But I could help with your problem

P.S.#2 My personal Hac has similar spec (ASUS FX504) running OC 0.9.1 and ventura 13.3.1 
My wifi&BT (Intel Wireless-AC 9560) (same card) is working just fine but it took a bit long time to boot (Cause by Bluetooth Kext) But it work flawlessly 

My suggestion is buy a native wifi card if you are daily drive your Hac  

Sorry for (Very)late reply I just log back to github

**Comment by billabongbruno** [comment]
> What is your smbios info ? It should be MacBookPro14,3 if you use my efi because of the `USBMap.kext` is set target to this smbios
> 
> and for the wifi you should use https://github.com/OpenIntelWireless/itlwm/releases , v.2.2.0 alpha If you update your efi and kext with ocat https://github.com/ic005k/OCAuxiliaryTools Do not tick on `AirportItlwm.kext` It will download v.2.1.0 instead which is not work on ventura
> 
> For Bluetooth follow https://openintelwireless.github.io/IntelBluetoothFirmware/FAQ.html#what-does-this-kext-do In short Use IntelBTPatcher.kext & IntelBluetoothFirmware.kext + BlueToolFixup.kext do not put IntelBluetoothInjector.kext this will make a Bluetooth work but also took a system a bit long to boot
> 
> ofc "I just get a black screen and never even get to see verbose or OpenCore." Yeah That's a very common problem if you dl pre-build efi from the internet a laptop with the same model sometime in different region some component isn't the same I had got the same problem too in good old clover bootloader days
> 
> P.S.#1 I do not have the device/own the device This is my friend laptop But I could help with your problem
> 
> P.S.#2 My personal Hac has similar spec (ASUS FX504) running OC 0.9.1 and ventura 13.3.1 My wifi&BT (Intel Wireless-AC 9560) (same card) is working just fine but it took a bit long time to boot (Cause by Bluetooth Kext) But it work flawlessly
> 
> My suggestion is buy a native wifi card if you are daily drive your Hac
> 
> Sorry for (Very)late reply I just log back to github

Hey there.

Thank you for your reply.
It's perfectly alright that you took your time to reply, no worries.

Regarding the issue:
I believe I didn't quite make myself clear.
I did have the exact same hardware as stated and generated my personal SMBIOS with MacBook Pro 14,3 details.
It simply didn't boot.

Regardless, it is a personal issue, as I run a multiple boot environment (Kali, Ubuntu, Windows, macOS and ChromeOS).

I reverted back to my personal EFI and I got Bluetooth working again after fiddling with it under Kali (because it got disabled there too when I updated Kali's kernel, for some reason).
After a successful boot into macOS with BT working, I cycled trough all OS'es (except Kali) and then back to macOS and had BT working under all of them and then again in macOS.
However, after booting back into Kali, I did have BT under it, but after rebooting (and shutting down and all that), I could not get BT working under macOS again.

Kali does something to the BT module that prevents it from starting under macOS.
I am unsure as to what that is and I got tired of debugging it.

I went ahead and got a BCM94360NG Fenvi card and now I have BT working natively under macOS and under all other OS'es.

I bounce around between all the before mentioned OS'es on a daily basis, so for me, it was worth the investment.

I tried my best to keep the stock hardware as much as I could, but to be honest, under Ventura, the Wifi was indeed taking a veeeeeeeeery long time to boot up and that kind of annoyed me too.

With the native BCM94360NG Fenvi card, it just works as soon as I log in, which is great.

This also allowed to me cleanup my EFI folder quite a bit, as I only use the BlueToolFixUp.kext now.
For Fenvi cards, no other kexts are needed, which is great.

So, in summary, I gave up on the Intel AC-9560 and got a Broadcom 94360NG Fenvi card.

lol

Anyway, thank you for your help and for the time you took to write your proposed solution details.

**Comment by pnapt** [comment]
TBH Me neither understand about black screen problem but 
It always happen when I download a pre-build efi from other people lol

Thanks for reply, Have a good day.

**Comment by Brexx69** [comment]
> > What is your smbios info ? It should be MacBookPro14,3 if you use my efi because of the `USBMap.kext` is set target to this smbios
> > and for the wifi you should use https://github.com/OpenIntelWireless/itlwm/releases , v.2.2.0 alpha If you update your efi and kext with ocat https://github.com/ic005k/OCAuxiliaryTools Do not tick on `AirportItlwm.kext` It will download v.2.1.0 instead which is not work on ventura
> > For Bluetooth follow https://openintelwireless.github.io/IntelBluetoothFirmware/FAQ.html#what-does-this-kext-do In short Use IntelBTPatcher.kext & IntelBluetoothFirmware.kext + BlueToolFixup.kext do not put IntelBluetoothInjector.kext this will make a Bluetooth work but also took a system a bit long to boot
> > ofc "I just get a black screen and never even get to see verbose or OpenCore." Yeah That's a very common problem if you dl pre-build efi from the internet a laptop with the same model sometime in different region some component isn't the same I had got the same problem too in good old clover bootloader days
> > P.S.#1 I do not have the device/own the device This is my friend laptop But I could help with your problem
> > P.S.#2 My personal Hac has similar spec (ASUS FX504) running OC 0.9.1 and ventura 13.3.1 My wifi&BT (Intel Wireless-AC 9560) (same card) is working just fine but it took a bit long time to boot (Cause by Bluetooth Kext) But it work flawlessly
> > My suggestion is buy a native wifi card if you are daily drive your Hac
> > Sorry for (Very)late reply I just log back to github
> 
> Hey there.
> 
> Thank you for your reply. It's perfectly alright that you took your time to reply, no worries.
> 
> Regarding the issue: I believe I didn't quite make myself clear. I did have the exact same hardware as stated and generated my personal SMBIOS with MacBook Pro 14,3 details. It simply didn't boot.
> 
> Regardless, it is a personal issue, as I run a multiple boot environment (Kali, Ubuntu, Windows, macOS and ChromeOS).
> 
> I reverted back to my personal EFI and I got Bluetooth working again after fiddling with it under Kali (because it got disabled there too when I updated Kali's kernel, for some reason). After a successful boot into macOS with BT working, I cycled trough all OS'es (except Kali) and then back to macOS and had BT working under all of them and then again in macOS. However, after booting back into Kali, I did have BT under it, but after rebooting (and shutting down and all that), I could not get BT working under macOS again.
> 
> Kali does something to the BT module that prevents it from starting under macOS. I am unsure as to what that is and I got tired of debugging it.
> 
> I went ahead and got a BCM94360NG Fenvi card and now I have BT working natively under macOS and under all other OS'es.
> 
> I bounce around between all the before mentioned OS'es on a daily basis, so for me, it was worth the investment.
> 
> I tried my best to keep the stock hardware as much as I could, but to be honest, under Ventura, the Wifi was indeed taking a veeeeeeeeery long time to boot up and that kind of annoyed me too.
> 
> With the native BCM94360NG Fenvi card, it just works as soon as I log in, which is great.
> 
> This also allowed to me cleanup my EFI folder quite a bit, as I only use the BlueToolFixUp.kext now. For Fenvi cards, no other kexts are needed, which is great.
> 
> So, in summary, I gave up on the Intel AC-9560 and got a Broadcom 94360NG Fenvi card.
> 
> lol
> 
> Anyway, thank you for your help and for the time you took to write your proposed solution details.

Hi
I have a problem with the bluetooth and I tried everything, so I thought to install this card, but when I search on amazon I see multiple article of this card but with different brands like: Wresetly, Exoggo, Bassulouda. Which one did you buy and how can I now which one is genuine?
